nibabel/streamlines/tests/test_array_sequence.py

```diff
@@ -262,7 +271,9 @@ def test_arraysequence_getitem(self):
                        for i, keep in enumerate(selection) if keep])
 
         # Test invalid indexing
-        assert_raises(TypeError, SEQ_DATA['seq'].__getitem__, 'abc')
+        with pytest.raises(TypeError):
+            SEQ_DATA['seq'].__getitem__('abc')
```

Comment:
`__getitem__` is called when indexing.

```suggestion
            SEQ_DATA['seq']['abc']
```

Reply:
:man_facepalming: